irohad/multi_sig_transactions/impl/mst_processor_impl.cpp

```diff
@@ -32,6 +32,13 @@ namespace iroha {
                     });
     }
   }
+  void shareState(
```

Comment:
Add documentation for the function and maybe rename it or previous shareState. Now it isn't obvious that they are doing the not same job.